src/main/groovy/com/smartbear/swagger/Swagger2TestCaseGenerator.groovy

```diff
@@ -0,0 +1,60 @@
+package com.smartbear.swagger
+
+import com.eviware.soapui.impl.rest.RestRequestInterface.HttpMethod
+import com.eviware.soapui.impl.rest.RestResource
+import com.eviware.soapui.impl.wsdl.WsdlProject
+import com.eviware.soapui.impl.wsdl.WsdlTestSuite
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.impl.wsdl.teststeps.registry.RestRequestStepFactory
+import com.eviware.soapui.model.testsuite.TestProperty
+import io.swagger.models.Operation
+import io.swagger.models.Path
+import io.swagger.models.parameters.AbstractSerializableParameter
+
+class Swagger2TestCaseGenerator {
+    static void generateTestCases(WsdlProject project, RestResource resource, Path path) {
+        WsdlTestSuite testSuite = createTestSuiteIfNotPresent(project)
+        WsdlTestCase testCase = testSuite.addNewTestCase("$resource.name-TestCase")
+        resource.restMethodList.each {
```

Comment:
I wanted to create it using the standard flow we have in Ready! API - where a request is added to test case. But sure I can change - what metadata are you referring to? We can still get that just like I get the parameter values- even if we create from already created requests.